app/src/main/java/com/readme/android/MainActivity.kt

```diff
@@ -1,11 +1,42 @@
 package com.readme.android
 
 import android.os.Bundle
+import androidx.viewpager2.widget.ViewPager2
+import com.readme.android.adapter.MainViewPagerAdapter
 import com.readme.android.core.base.BindingActivity
 import com.readme.android.databinding.ActivityMainBinding
+import com.readme.android.main.ui.home.HomeFragment
+import com.readme.android.main.ui.mypage.MyPageFragment
+import dagger.hilt.android.AndroidEntryPoint
 
+@AndroidEntryPoint
 class MainActivity : BindingActivity<ActivityMainBinding>(R.layout.activity_main) {
+    private lateinit var mainViewPagerAdapter: MainViewPagerAdapter
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        initAdapter()
+        syncVpWithBottomNav()
+    }
+
+    private fun initAdapter() {
+        binding.vpMain.adapter = MainViewPagerAdapter(this).also { mainViewPagerAdapter = it }
+        mainViewPagerAdapter.fragments.addAll(listOf(HomeFragment(), MyPageFragment()))
```

Comment:
> 굳이굳이 캡슐화 원칙 지켜서 코드 짠다면.. 이렇게 짜도 괜춘할듯합니당..!

위 리뷰따라서 코드 변경한다면
```suggestion
        mainViewPagerAdapter.fragmentList = listOf(HomeFragment(), MyPageFragment())
```
이렇게만 표기해도 괜춘

Reply:
진촤 친절 다빈😎